designer/client/src/components/toolbars/search/AdvancedSearchFilters.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useMemo, useRef } from "react";
+import { Button, Box, Typography } from "@mui/material";
+import { useTranslation } from "react-i18next";
+import { SearchLabeledInput } from "../../sidePanels/SearchLabeledInput";
+import { SearchLabel } from "../../sidePanels/SearchLabel";
+
+const transformInput = (input: string, fieldName: string) => {
+    return input === "" ? "" : `${fieldName}:(${input})`;
+};
+
+function extractSimpleSearchQuery(text: string): string {
+    const regex = /(\w+):\(([^)]*)\)/g;
+    let match: RegExpExecArray | null;
+    let lastIndex = 0;
+
+    while ((match = regex.exec(text)) !== null) {
+        lastIndex = regex.lastIndex;
+    }
+
+    const rest = text.slice(lastIndex).trim();
+
+    return rest;
+}
+
+export function AdvancedSearchFilters({
```

Comment:
We should consider syncing advanced search filters, based on the simple search values, so:
1. When we clear a simple search that has advanced search queries, values in the form also should be cleared
2. When we type a query manually, it should be visible in the advanced search form 


https://github.com/user-attachments/assets/ebd3d730-83de-412e-be66-45b06a2860b3